backend/utilities/dateUtils.ts

```diff
@@ -44,3 +44,9 @@ export const getTodayForTZIgnoredUTC = (date: Date = new Date()): Date => {
     .startOf("day")
     .toDate();
 };
+
+export const isPast = (date: Date): boolean => {
+  return moment(moment(date).utc().format("YYYY-MM-DD")).isBefore(
```

Comment:
```suggestion
  return moment(moment(date).format("YYYY-MM-DD")).isBefore(
```

I don't think UTC is needed

Reply:
update: we don't need this function. Let's remove it.